### PR TITLE
Add targets

### LINKED ISSUE #178: Add Makefile target to process results and plot charts

### PR DESCRIPTION
**Pull request by snim2**
Fixes #178 

Needs squashing.

### CODE REVIEW

**Comment by ltratt** [comment]
I'm not sure we want to hardcode the window size in here, at least not at this point. One way of doing things is to say something like `WINDOW_SIZE ?= 200` and then the user can override that with `make WINDOW_SIZE=100`.

**Comment by snim2** [comment]
Ready to squash?

**Comment by ltratt** [comment]
There are still hard-coded references to "200" left in the Makefile, which should depend on `WINDOW_SIZE` (I think).

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,10 @@
 PYTHON ?= python2.7
 PWD != pwd
 UNAME != uname
+WINDOW_SIZE ?= 200
+OUTLIER_THRESHOLD ?= 8
+PLOTS_NO_CPTS = plots_w200.pdf
```
Comment (vext01) [inline Makefile]: `PLOTS_NO_CPTS = plots_w${WINDOW_SIZE}.pdf`?

**Review by vext01 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (vext01) [reply Makefile]: And similar on the next line too.

**Comment by snim2** [comment]
OK, I think that is fixed.

**Comment by ltratt** [comment]
Yep, can be squashed, then I can merge.

**Comment by snim2** [comment]
Splat.

**Review by vext01 — APPROVED** [review]

**Comment by snim2** [comment]
Thanks!

**Pull request merged**